astropy/modeling/tests/test_constraints.py

```diff
@@ -521,16 +521,3 @@ def test_2d_model():
         assert_allclose(m.parameters, p2.parameters, rtol=0.05)
         m = fitter(p2, x, y, z + 2 * n, weights=None)
         assert_allclose(m.parameters, p2.parameters, rtol=0.05)
-
-
-def test_prior_posterior():
```

Comment:
Why was this removed?

Reply:
`prior` and `posterior` were originally defined as constraints because `Parameter`, being a descriptor, could not store data/attributes. The intention was to remove these two "constraints" and let users set them as attributes when they need them. I guess we could still keep some knd of test that this works.